api/src/main/java/com/codeforcommunity/dto/team/TeamDataResponse.java

```diff
@@ -5,64 +5,57 @@
 import java.util.List;
 
 public class TeamDataResponse {
-  private final Integer id;
```

Comment:
Didn't want to return the goals with the GetAllTeams route. There is now a TeamDataResponse that does not have goals, and a TeamGoalsDataResponse that extends TeamDataResponse that includes goals.